spec/spec_helper.rb

```diff
@@ -35,12 +35,6 @@ module Shell
 require "webmock/rspec"
 
 require "chef"
-require "chef/knife"
-
-Dir["lib/chef/knife/**/*.rb"]
```

Comment:
If we don't load them, we can't hide that we're accidentally referencing them.

Reply:
oh sure, this is crazy but more or less makes sense.